src/main/java/net/helpscout/api/utils/ParamsUtils.java

```diff
@@ -66,6 +69,19 @@ public static String setFields(String url, List<String> fields) {
         return params;
     }
 
+    public static Map<String, String> getSearchParamsMap(String query, String sortField, String sortOrder, Integer page) throws ApiException {
+        Map<String, String> params = new HashMap<String, String>();
+        params.put("query", query);
+        if (sortField != null && sortField.trim().length() > 0) {
```

Comment:
You could probably just insert everything to the `params` map and the remove all empty entries at once.